heltour/tournament/admin.py

```diff
@@ -661,7 +661,9 @@ def player_info_view(self, request, object_id, player_name):
             'league': season.league,
             'player': season_player.player,
             'reg': reg,
-            'has_played_20_games': has_played_20_games
+            'has_played_20_games': has_played_20_games,
+            'date_created': str(sp.registration.date_created or sp.player.date_created),
+            'friends': sp.registration.friends or None
```

Comment:
This also uses the date_created on the seasonplayer directly, not on the Player record which might be quite old.